### ISSUE TITLE
Can not store cars, nor can I access impound menu.

### ISSUE THREAD

**Issue by twistedcinema** [HumanTree92/esx_eden_garage#7]
I am looking for a little bit of help with this. I have set everything up correctly. I can pull a vehicle out of the garage but I can not store the same vehicle. I go to the impound and press E to open the menu and it goes away. No menu comes up.

**Comment by HumanTree92** [comment]
If you don't mind me asking is the server Linux or Windows?

**Comment by twistedcinema** [comment]
Windows

**Comment by HumanTree92** [comment]
Yeah i have no clue. I am pretty sure its Windows that is causing your problem. Had 2 other people try this on a windows server & had problems. But when they tried them on a Linux Server with the same code & nothing changed it all worked out just fine.

**Comment by twistedcinema** [comment]
Okay so I cant pull cars out with yours. I was using a different one but when I got o pull out a car it tells me this "[MySQL] An error happens on MySQL for query "UPDATE owned_vehicles SET state =@state WHERE plate=@plate {=}": Unknown column 'plate' in 'where clause'

**Comment by twistedcinema** [comment]
I have one version of eden garages right now which works. There is a red, yellow and blue circle. You get your car out with the yellow circle and the blue is where it spawns and the red is where you park the vehicle to store it. Im just trying to find a way people can buy their vehicle back if it gets impounded or stolen.

**Comment by HumanTree92** [comment]
Sounds like you are not using the new plate system correctly. I would make sure you have updated you DB to the new plate system. Follow this if you need help https://forum.fivem.net/t/release-esx-scripts-more/112491/51?u=humantree92

**Comment by twistedcinema** [comment]
I have read over the plate system and changed my database to look like that. when i purchase my car, I can store it. Then I have to add the plate to the database for it to work. But it doesnt update the state of the vehicle. the two issues I am having with it, The plate is not added to the plate section and the state of vehicle does not update. I am willing to pay for help with setting this up. Ive been stuck on it for about a week now. Thanks !

**Comment by SLuuuuuuG** [comment]
This has been working perfectly for me.    Wonderful release.   Every once in a while a players car will show that it's in the impound...they goto the impound and it says it's in the garage.     I cannot replicate the problem myself.   I have seen the issues with other players

**Comment by twistedcinema** [comment]
I have got it working for the most part. Only issue is with the impound. If a player has a vehicle in the pound and they pull a different vehicle from garage and store it, it adds that to the pound as well

**Comment by HumanTree92** [comment]
Yes Currently use his Garage if you have to. However tho i am re-writing the garage script. It won't use state of vehicle.
